modules/core/quick/tests/src/Kernel/QuickStringTest.php

```diff
@@ -72,4 +87,49 @@ public function testPrioritizedString() {
     $this->assertEquals('Foo B… Baz', $name);
   }
 
+  /**
+   * Test assetNamesSummary() method.
+   */
+  public function testAssetNamesSummary() {
+
+    // Create a test asset type.
+    $asset_type = AssetType::create([
+      'id' => 'test',
+      'label' => 'Test',
+      'workflow' => 'asset_default',
+    ]);
+    $asset_type->save();
+
+    // Create 10 assets with randomly generated names.
+    $assets = [];
+    for ($i = 0; $i < 10; $i++) {
+      $asset = Asset::create([
+        'name' => $this->randomString(),
+        'type' => 'test',
+        'status' => 'active',
+      ]);
+      $asset->save();
+      $assets[] = $asset;
+    }
+
+    // Test default with a cutoff of 3.
+    $expected = $assets[0]->label() . ', ' . $assets[1]->label() . ', ' . $assets[2]->label() . ' (+ 7 more)';
+    $name_summary = $this->assetNamesSummary($assets);
+    $this->assertEquals($expected, $name_summary);
+
+    // Test with a cutoff of 1.
+    $expected = $assets[0]->label() . ' (+ 9 more)';
+    $name_summary = $this->assetNamesSummary($assets, 1);
+    $this->assertEquals($expected, $name_summary);
+
+    // Test default with a cutoff of 0.
```

Comment:
This comment is inaccurate (in calling zero the default) right?

Reply:
Ah good catch @symbioquine - I'll push a fixup commit.